usr/share/git/ReleaseNotes.css

```diff
@@ -0,0 +1,373 @@
+html {
```

Comment:
I believe that you took care of this with the merge of PR 113 or 117--but you are correct. I copied it for "safety" to make sure everything worked as I intended, with the intention of deleting the original. Needless to say, I forgot to delete the original. I'll take care of this when I implement your suggestion for a "render release notes" script.